### PR TITLE
Faulty selection collection.

### PR DESCRIPTION
**Pull request by tokejepsen**

### CODE REVIEW

File: pyblish_nukestudio/plugins/collect_selection.py [hunk 1]
```diff
@@ -1,15 +1,13 @@
 import pyblish.api
 
-import hiero
 
-
-class CollectSelection(pyblish.api.Contextplugin):
+class CollectSelection(pyblish.api.ContextPlugin):
     """Inject the active project into context"""
```
Comment (mottosso) [inline pyblish_nukestudio/plugins/collect_selection.py]: Docstring seems off

File: pyblish_nukestudio/plugins/collect_selection.py [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-    version = (0, 1, 0)
     order = pyblish.api.CollectorOrder - 0.1
 
     def process(self, context):
+        import hiero
 
-        if hasattr(hiero, 'selection'):
-            context.set_data('selection', value=hiero.selection)
+        if hasattr(hiero, "selection"):
+            context.data["selection"] = hiero.selection
```
Comment (mottosso) [inline pyblish_nukestudio/plugins/collect_selection.py:13]: I'm not familiar with Nuke Studio, but should publishing really be based on the active selection? I figure it'd make it tricky to publish non-interactively, if that's even possible.

File: pyblish_nukestudio/plugins/collect_selection.py (hunk 1 above)
Comment (tokejepsen) [reply pyblish_nukestudio/plugins/collect_selection.py]: Good point! Fixed.

File: pyblish_nukestudio/plugins/collect_selection.py (hunk 2 above)
Comment (tokejepsen) [reply pyblish_nukestudio/plugins/collect_selection.py:13]: When you export anything from NukeStudio, you need to select it.

You may be right about that we might be encouraging this interactive publishing, though think that might up each developer to decide how to handle that. The developer could for example tag the selection for later non-interactive publishing.
From the users perspective they would be able to do same thing as they are used to.

**Comment by tokejepsen** [comment]
@mottosso think we can merge this?

**Comment by mottosso** [comment]
Of course, go right ahead. Let me know if you need permissions set or anything.

**Pull request merged**